app/models/controlled_vocabulary.rb

```diff
@@ -324,11 +324,35 @@ def self.authority_config
       @notice_authority_config ||= YAML.safe_load(File.read(Rails.root.join("config", "authorities", "notices.yml")), [Symbol])
     end
 
+    # Return the notice for a given resource.
+    # @return [Term]
+    def for(resource)
+      if resource.try(:primary_imported_metadata)&.content_warning.present?
+        NoticeTerm.new(value: "specific_harmful_content", definition: resource.primary_imported_metadata.content_warning)
```

Comment:
It's a little confusing that it's loaded in as the definition here, then on line 349 (via the method on 353) it's referred to as the message, while actually getting the full "definition" from the i18n file.

I wonder if we could use "message" in the i18n file and keep the use of "definition" just for the `specific_harmful_content` terms? Or maybe there's some other way to greater clarity here.